### PR TITLE
GraphiQL packages upgrade to latest

### PR DESCRIPTION
**Pull request by rodrigofeijao**
## Description

Upgrade GraphiQL packages to latest:

- `graphiql@3.0.9`
- `@graphiql/plugin-explorer@1.0.2`

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Upgrade GraphiQL packages to latest:

- `graphiql@3.0.9`
- `@graphiql/plugin-explorer@1.0.2`



Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Rodrigo Feijao for the PR 👏

Get it here 👉 https://beta.strawberry.rocks/release/(next)
```

File: strawberry/static/graphiql.html [hunk 1]
```diff
@@ -61,14 +61,14 @@
     <link
       crossorigin
       rel="stylesheet"
-      href="https://unpkg.com/graphiql@3.0.5/graphiql.min.css"
+      href="https://unpkg.com/graphiql@3.0.9/graphiql.min.css"
       integrity="sha384-yz3/sqpuplkA7msMo0FE4ekg0xdwdvZ8JX9MVZREsxipqjU4h8IRfmAMRcb1QpUy"
```
[the remark this answers is not in the document]
Comment (rodrigofeijao) [reply strawberry/static/graphiql.html:65]: good point! Let me update them. tks

[the remark this answers is not in the document]
Comment (rodrigofeijao) [reply strawberry/static/graphiql.html:65]: Apparently some of them use the same hash. 🤷 
Cheked using: https://www.srihash.org/

[the remark this answers is not in the document]
Comment (patrick91) [reply strawberry/static/graphiql.html:65]: ah, didn't know about that tool! thanks :) I'll check if it all works and then I'll release :)

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Release type: minor
+
+Upgrade GraphiQL packages to latest:
+
+- `graphiql@3.0.9`
+- `@graphiql/plugin-explorer@1.0.2`
```
Comment (patrick91) [inline RELEASE.md:6]:
```suggestion
This release updates the GraphiQL packages to their latest versions:

- `graphiql@3.0.9`
- `@graphiql/plugin-explorer@1.0.2`
```

**Review by patrick91 — APPROVED** [review]

**Comment by patrick91** [comment]
I'll fix the lint errors and then merge, those are not related to this PR

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**